### DISCUSSION TITLE
abgx360gui compile issues

### DISCUSSION THREAD

**Discussion by y700** [BakasuraRCE/abgx360#6] [Q&A]
your readme states to ./configure then make and then make install, however the configure files and make files are not availble from the git repo or zip file and as such this means the gui can not be compiled, can you assit with this compile platform Ubuntu 22

**Comment by BakasuraRCE** [comment]
Your are right, the readme is old, these isntructions are for the original source that is here: https://github.com/BakasuraRCE/abgx360/releases/download/v1.0.6-and-v1.0.2/linux_abgx360gui-1.0.2.tar.gz and here https://github.com/BakasuraRCE/abgx360/releases/download/v1.0.6-and-v1.0.2/linux_abgx360-1.0.6.tar.gz

If you are looking for the version I was working on with various improvements and changes, you should go to these repositories instead and build them using CMake. I recommend using CLion for this; it will assist you throughout the process:
https://github.com/BakasuraRCE/abgx360-reloaded
https://github.com/BakasuraRCE/abgx360gui-reloaded
